src/state-keys.ts

```diff
@@ -0,0 +1 @@
+export const VOLTA_HOME = 'VOLTA_HOME';
```

Comment:
I'd rather just use the string where we need it (instead of using this constant).